modules/skybrowser/skybrowsermodule.cpp

```diff
@@ -0,0 +1,692 @@
+        "the mouse or not."
+    };
+    
+    constexpr const openspace::properties::Property::PropertyInfo AllowRotationInfo =
+    {
+        "AllowCameraRotation",
+        "Allow Camera Rotation",
+        "Toggles if the camera should rotate to look at the sky target if it is going "
+        "outside of the current field of view."
+    };
+
+    struct [[codegen::Dictionary(SkyBrowserModule)]] Parameters {
+        // [[codegen::verbatim(AllowInteractionInfo.description)]]
+        std::optional<bool> allowMouseInteraction;
+
+        // [[codegen::verbatim(AllowRotationInfo.description)]]
+        std::optional<bool> allowCameraRotation;
+    };
+    
+    #include "skybrowsermodule_codegen.cpp"
+} // namespace
+
+namespace openspace {
+    scripting::LuaLibrary SkyBrowserModule::luaLibrary() const {
+        scripting::LuaLibrary res;
+        res.name = "skybrowser";
+        res.functions = {
+            {
+                "getListOfImages",
+                &skybrowser::luascriptfunctions::getListOfImages,
+                "",
+                "Returns a list of all the loaded AAS WorldWide Telescope images that "
+                "has been loaded."
+            }, 
+            {
+                "setHoverCircle",
+                &skybrowser::luascriptfunctions::setHoverCircle,
+                "string",
+                "Takes an identifier to a screen space renderable and adds it to the "
+                "module."
+            },
+            {
+                "moveCircleToHoverImage",
+                &skybrowser::luascriptfunctions::moveCircleToHoverImage,
+                "int",
+                "Moves the hover circle to the coordinate specified by the image index."
+            },
+            {
+                "disableHoverCircle",
+                &skybrowser::luascriptfunctions::disableHoverCircle,
+                "",
+                "Disables the hover circle, if there is one added to the sky browser "
+                "module. "
+            },
+            {
+                "loadImagesToWWT",
+                &skybrowser::luascriptfunctions::loadImagesToWWT,
+                "string",
+                "Takes an identifier to a sky browser or target and loads the WWT image "
+                "collection to that browser."
+            },
+            {
+                "selectImage",
+                &skybrowser::luascriptfunctions::selectImage,
+                "int",
+                "Takes an index to an image and selects that image in the currently "
+                "selected sky browser."
+            }, 
+            {
+                "removeSelectedImageInBrowser",
+                &skybrowser::luascriptfunctions::removeSelectedImageInBrowser,
+                "string, int",
+                "Takes an identifier to a sky browser or target and an index to an "
+                "image. Removes that image from that sky browser."
+            },
+            {
+                "adjustCamera",
```

Comment:
This function name is not really reflecting what the description says that it does, imo. Maybe give it a more specific name?